cpp-client/tools/aiaa/aiaa-dextra3d.cpp

```diff
@@ -72,8 +76,8 @@ int main(int argc, char **argv) {
     std::cerr << "Pointset is empty\n";
     return -1;
   }
-  if (inputImageFile.empty()) {
-    std::cerr << "Input Image file is missing\n";
+  if (inputImageFile.empty() && (!preProcess && sessionId.empty())) {
```

Comment:
why !preProcess here?
If input image is empty and sessionId is empty, then no matter the value of pre-process, we should return -1?
What do I miss

Reply:
if preProcess is true.. => inputImageFile can't be empty
else => (inputImageFile or sessionId should be present)

but lets make the condition simple.. fixed the condition